### ISSUE TITLE
pressing grave key closes application

### ISSUE THREAD

**Issue by h-b** [BestImageViewer/geeqie#979]
### ISSUE TYPE
* Bug Report

### GEEQIE VERSION

```
Geeqie 1.7.2
```
### OS / DISTRIBUTION

Operating system:     Zorin OS 16.1
Ubuntu codename:      focal
Debian version:       bullseye/sid

### SUMMARY

The keyboard shortcut "grave" (key above tab) to toggle menu bars just closes geeqie. I hided bars via the menu, but there is no option to show them in the context menu, so it is essential that the shortcut works. As a result I am unable to show bars again (even after reinstallation).

In version 1.5.1 this works. I switched back and forth between these two versions to make sure. Unfortunately 1.5.1 is missing some features I need.

One other shortcut is affected. To help debugging, I tried all that are listed in the context menu:
- **working:** = _ Z z X x Ctrl+v Ctrl+c Ctrl+m Ctrl+r Delete Shift+Delete S v
- **not working (application is closed):** grave Ctrl+h

(Side remark: In this list I use uppercase and lowercase exactly like it works for me. For example, typing 'V' as documented does not switch to full screen for me. Only 'v'.)

### STEPS TO REPRODUCE

While bars are hidden, press the grave key or Ctrl+h. (Currently I cannot test if it happens when the bars are shown, because I cannot unhide them.)

**Comment by caclark** [comment]
>  (Currently I cannot test if it happens when the bars are shown, because I cannot unhide them.)

Close Geeqie and open $HOME/.config/geeqierc.xml in an editor.
In the `<layout>` section there is an entry for `tools_hidden`.
Change this to `false`.

The right-click menu item `Hide File List` also controls this function.

The reason for your problem i do not yet know.

**Comment by h-b** [comment]
Thanks for the info. I changed the flag in my `geeqierc.xml` and the bars are shown again, which is great.

Just to make sure: When I use the menu item `Hide File List` to hide them and then use it again to unhide them, the app closes (just as described in the report).

Maybe the reason of the issue is not the version, but the way of installation:

- Version `1.5.1` (where the issue does not occur) I installed via the Debian package manager
- Version `1.7.2` (where the issue does occur) I installed via flathub (so the corresponding geeqierc.xml is in `$HOME/.var/app/org.geeqie.Geeqie/config/geeqie/geeqierc.xml`)

There are another two things which might be helpful. I tested it several times. Apparently, you write the `geeqierc.xml` only to disk when the app is closed by the user.

1. Since the flag is only written to disk when the user closes the app, you are only in trouble if you actually close the app with hidden bars. If you switch back and forth between the two states without closing geeqie, the bars will always show up when you open geeqie after it closed itself because `tools_hidden = "true"` was never written to disk. Perhaps there are users who have experienced the same thing, but have had no problems because they unhid the bars before closing.
2. Hiding the bars via the menu item always closes the app (as described). But sometimes, in the minority of cases, `tools_hidden = "true"` is actually written to disk, so that when geeqie is reopened after it has closed itself, the bars are actually shown. I never observed this when I used the shortcut (that's why I couldn't unhide them before I learned about the menu item).

geeqie offers a log window including debug level. Unfortunately, (1) it is not accessible when the bars are hidden and (2) it closes when the app closes. Is there a corresponding log file that contains the same info?
